modules/guides/partials/javascript-udfs/start-cbq-shell-instructions.adoc

```diff
@@ -0,0 +1,6 @@
+////
```

Comment:
You don't need this partial every single time the users need to access cbq. Please take a look at the way the other how-to guides handle this.